src/agents/workflows/subgraphs/step1_preprocessing/vision_llm.py

```
@@ -45,104 +37,166 @@ class VisionProvider:
+                        "ocr": [
+                            {
+                                "page": 1,
+                                "ocr_text": raw_content,
+                                "blocks": [],
+                            }
+                        ],
+                        "image_caption": [],
+                    }
+        except Exception:
+            preview = raw_content[:200] if "raw_content" in locals() else ""
+            logger.error(f"OpenRouter Gemini 응답 원본 확인: {preview}...")
```

Ruff (0.14.13)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)